addons/hr_expense/tests/test_expenses_mail_import.py

```diff
@@ -212,3 +213,18 @@ def assertParsedValues(subject, currencies, exp_description, exp_amount, exp_pro
             self.product_a,
             self.company_data['currency'],
         )
+
+
+    def test_import_expense_from_mail_get_default_expense_sheet_values_errors(self):
+        # error no product
```

Comment:
This comment doesn't tell me much about what the test is really testing, could you clarify it ?